alpha/scripts/utils/updateEntrySatutsToError.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+require_once (dirname(__FILE__) . '/../bootstrap.php');
+
+if ($argc < 2)
+{
+	echo PHP_EOL . ' ---- Update Pending Clipping Entries ---- ' . PHP_EOL;
+	echo ' Execute: php ' . $argv[0] . ' [ /path/to/entries_id_list || entryId_1,entryId_2,.. || entry_id ] [realrun / dryrun]' . PHP_EOL;
+	die(' Error: missing entry_ids file, csv or single entry ' . PHP_EOL . PHP_EOL);
```

Comment:
instead of logging then die just change the die message to something like
die("Usage: php updateEntrySatutsToError.php < /path/to/entries_id_list || entryId_1,entryId_2,.. || entry_id > <realrun / dryrun>"."\n");